.github/workflows/nodejs.yml

```diff
@@ -4,32 +4,71 @@ on:
   push:
     branches:
     - develop
+    - test
+
   pull_request:
 
+  release:
+    types: [published]
+
 jobs:
-  build:
-    runs-on: 'ubuntu-latest'
+  release:
+    runs-on: ubuntu-latest
+
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/v')
 
     steps:
-      - name: Setup Node.js environment
-        uses: actions/setup-node@v3
+      - name: 'Checkout Project'
+        uses: 'actions/checkout@v3'
         with:
-          node-version: '16.15'
+          fetch-depth: 1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Build docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          push: true
+          tags: vnguyen/openbeta-graph-api:latest
```

Comment:
Should we use Git tag instead of latest?  Example: `vnguyen/openbeta-graph-api:v1.0.2`